openfe/tests/setup/conftest.py

```diff
@@ -43,14 +42,14 @@ def other_mapping():
     molA = SmallMoleculeComponent(Chem.MolFromSmiles('CCO'))
     molB = SmallMoleculeComponent(Chem.MolFromSmiles('CC'))
 
-    m = LigandAtomMapping(molA, molB, mol1_to_mol2={0: 0, 2: 1})
+    m = LigandAtomMapping(molA, molB, molA_to_molB={0: 0, 2: 1})
 
     return m
 
 
 @pytest.fixture(scope='session')
 def lomap_basic_test_files():
-    # a dict of {filenames.strip(mol2): SmallMoleculeComponent} for a simple
+    # a dict of {filenames.strip(molB): SmallMoleculeComponent} for a simple
```

Comment:
```suggestion
    # a dict of {filenames.strip(mol2): SmallMoleculeComponent} for a simple
```
🤣

Reply:
Yeah ok well at least this is good evidence of reason for the change :)

Reply:
Yeah 100%, I think I've caught them all now.